beacon-chain/blockchain/forkchoice/process_attestation.go

```diff
@@ -120,10 +120,9 @@ func (s *Store) OnAttestation(ctx context.Context, a *ethpb.Attestation) (uint64
 			return 0, err
 		}
 		delete(s.attsQueue, root)
-	}
-
-	if err := s.saveNewAttestation(ctx, a); err != nil {
-		return 0, err
+		if err := s.saveNewAttestation(ctx, a); err != nil {
```

Comment:
inserted by mistake?

Reply:
no we need to save the aggregated attestation, not the unaggregated ones, so its put in 
the attestation queue loop

Reply:
got it. my mistake